README.md

```diff
@@ -1,70 +1,59 @@
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## ARCHITECTURE
+
+### Context: 
+  #### Data Context
+    NECESSARY for fetching and setting current data:
+    - [ show, setShows ] / fetch and set all shows by year.
+    - [ isLoading, setIsLoading ] 
+    = [ currentSong, setCurrentSong ] / fetch current song details to play track from two components - audio player component (next button) and track on click event.
+    - [ tracks, setTracks ] / get all tracks by show
+    OPTIONAL:
+    - [ year, setYear ]  / use year for styling logo/header
+
+    ```<RecipeContext.Provider value= {
+      updateTracks,
+      updateShows,
+      setCurrentSong
+    }>
+    ```
+
+  ```Note: use effect will be used on most individual components with router: 
+    useEffect(() => {
+      updateShows(url${this.params.key})
+      // or updateTracks(url${this.params.key})
+    }, [])
+  ```
+
+  #### Values to be passed into child components:
+    - updateShows(year)
+    - updateTracks(show)
+    - setCurrentSong //get details 
+
+### Router:
+```
+In app file return (
+  <DataProvider>
+    <Route render Dashboard/YearDisplay >
+    <Route render ShowDisplay >
+    <Route render TrackDisplay >
+  </DataProvider>
+)
+```
+
+### Component Folder:
+  - Header
```

Comment:
Header and footer may be able to be in a 'Page' component. 
This would allow our App.js would be pretty bare bones holding routes and switch.